craft_application/services/provider.py

```diff
@@ -47,7 +47,10 @@
     from craft_application.services import ServiceFactory
 
 
-DEFAULT_FORWARD_ENVIRONMENT_VARIABLES: Iterable[str] = ()
+DEFAULT_FORWARD_ENVIRONMENT_VARIABLES: Iterable[str] = (
```

Comment:
This workaround for 5.2 conflicts with the ConfigService forwarding in 5.3+. I think the appropriate thing would be to remove the workaround in a commit but still merge the changelog.